api/database/boltdb/db.go

```diff
@@ -2,6 +2,7 @@ package boltdb
 
 import (
 	"encoding/binary"
+	e "errors"
```

Comment:
mmm, can we keep this one named errors, and alias the (annoying) `dataservices/errors` instead?